docs/configuration.md

```diff
@@ -227,7 +227,7 @@ Builds support any named attributes that can be referenced by subsampling scheme
 ### min_date
 * type: float or string
 * description: Earliest date to estimate frequencies for. Dates can be numeric floating point values (e.g., `2019.74`) or ISO 8601-style strings (e.g., `2019-10-01`).
-* default: `2020.0`
+* default: with value supplied, defaults to 1 year before present
```

Comment:
Do you mean _without_ value supplied?

Reply:
Yes! I'll fix.

Reply:
Done in dedf9456eb442a7babb9c9640cc034ca226adfc5 and force pushed to branch.